keras/src/backend/torch/core.py

```diff
@@ -159,18 +159,22 @@ def __eq__(self, other):
 def convert_to_tensor(x, dtype=None, sparse=None):
     if sparse:
         raise ValueError("`sparse=True` is not supported with torch backend")
+    if type(x) is Variable:
+        # We cannot use `isinstance(x, Variable)` due to the failure of
```

Comment:
Hi Hongyu,

How do you reproduce this issue? I need to change this line and I want to make sure I don't create a regression here.

Thanks!

Reply:
Hi @hertschuh 

It failed to compile with the torch backend initially, but it seems to be working now.
(I've noticed that the CI passed in your PR)

Reply:
Great, if it used to happen with the CI tests, then it's passing now, so it looks like it's fixed.

Thanks!